tests/unit/src/test/scala/tests/DebugDiscoverySuite.scala

```diff
@@ -136,6 +141,13 @@ class DebugDiscoverySuite
   }
 
   test("run-file-test") {
+    val envFile: Path =
+      Files.write(
```

Comment:
I noticed in some tests that `cleanWorkspace` is called. Should we take care to call this as a finalizer here?

Reply:
This is mostly for local development, if the test changes somehow we might get leftover files, which disturb the results. You can add cleanWorkspace before, but it's fine not to

Reply:
Alright, I will leave it as is for now but let me know if you feel strongly otherwise and I can add it.